tests/main/views/test_direct_award.py

```diff
@@ -108,6 +116,11 @@ def test_save_search_submit_success(self):
         assert res.status_code == 303
         assert urlparse(res.location).path == '/buyers/direct-award/g-cloud/projects/1'
 
+    def test_save_existing_search_submit_success(self):
+        res = self._create_project('some name " foo bar \u2016', '1')
+        assert res.status_code == 303
+        assert urlparse(res.location).path == '/buyers/direct-award/g-cloud/projects/1'
```

Comment:
Interestingly I think actually this case should be an error: the user shouldn't submit anything in the 'name' field if they are choosing an existing project?

(So there should actually be two test cases here, and an additional validation we forgot about?)

Reply:
oh ye that is true

Reply:
updated the test to not send name